src/dxvk/imgui/dxvk_imgui.cpp

```diff
@@ -1339,12 +1438,16 @@ namespace dxvk {
 
     ImGui::Checkbox("Preserve discarded textures", &RtxOptions::Get()->keepTexturesForTaggingObject());
 
+    if (IMGUI_ADD_TOOLTIP(ImGui::CollapsingHeader("Step 0: Texture Selection", collapsingHeaderClosedFlags), "Select texture definitions for Remix")) {
+      showTextureSelectionGrid(ctx, "textures", numThumbnailsPerRow, thumbnailSize);
+    }
+
     if (IMGUI_ADD_TOOLTIP(ImGui::CollapsingHeader("Step 1: UI Textures", collapsingHeaderClosedFlags), RtxOptions::Get()->uiTexturesDescription())) {
-      showTextureSelectionGrid("uitextures", numThumbnailsPerRow, thumbnailSize, RtxOptions::Get()->uiTexturesRef());
+      showTextureSelectionGrid(ctx,"uitextures", numThumbnailsPerRow, thumbnailSize);
```

Comment:
I think we should deprecate these old texture lists.  Your solution is much nicer.  We can leave them around, and enable them on some RtxOption (so if anyone really misses them, they can be re-enabled) to be cleaned up at a later date maybe?

Reply:
Sure, that's a great solution!